Python/syntax_test_python_strings.py

```diff
@@ -430,7 +430,7 @@
 #      ^^^^ - constant.other.placeholder.python
 #            ^ invalid.illegal.unclosed-string.python
 '{0:%Y}-{0:%
-# ^^^^^^^^^^^ string.quoted.single.python
+# ^^^^^^^^^^ string.quoted.single.python
```

Comment:
just to check my understanding, does this change mean we no longer scope the `\n` as `string` (or `meta.string`)? If so, I'm worried it could cause problems when typing at EOL, for example completions are normally disabled inside strings.

Reply:
That's a side effect of switching to syntax version 2 as `meta_content_scope` is no longer applied to patterns (`illegal-string-end`) which set another context on stack. Dropping the test was the easier solution at that time.

Completions shouldn't be effected as ST4 correctly uses the scope of the char in front of the caret, but restoring the old scheme is an easy fix now that major refactorings with regards to version 2 have been completed.

Reply:
Done.